src/send.js

```diff
@@ -32,8 +35,14 @@ let sendDataPoints = async (configurer, options, payload) => {
       },
       encoding: null,
       body:     serialize(payload),
-      timeout:  30 * 1000,
-      retries:  5,
+      timeout:  90 * 1000,
+      retries:  (iter, error) => {
```

Comment:
Wait think you got the timing wrong.... I think the timeout is for sending including all retries... Oh, i guess it could be request timeout... Hmm cant remember...


But do we actually care? We record metrics for 5 min intervals... So even if it takes 10min, we'll only have two submissions in-flight...